src/templates/Resources/Medication.hbs

```diff
@@ -10,7 +10,7 @@
                 {{/with}}
             {{else}}
                 {{>DataType/CodeableConcept.hbs code=medication.code}},
-            {{/if}},             
+            {{/if}}    
```

Comment:
Actually, this comma should also be kept since #with contribution might be empty. So I guess mostly commas after {{/...}} should be good to have (unless some really complicated logic e.g. conditional key value).